src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2581,7 +2581,8 @@ def tag(self, tags, is_incremental=False):
         # please add the service type that needs to be requested with PATCH type here
         # for example: the properties of RecoveryServices/vaults must be filled, and a PUT request that passes back
         # to properties will fail due to the lack of properties, so the PATCH type should be used
-        need_patch_service = ['Microsoft.RecoveryServices/vaults', 'Microsoft.Resources/resourceGroups']
+        need_patch_service = ['Microsoft.RecoveryServices/vaults', 'Microsoft.Resources/resourceGroups',
+                              'Microsoft.ContainerRegistry/registries/webhooks']
```

Comment:
Can we add test for the two rps scenarios?

Reply:
Done